_rules/image-filename-as-accessible-name-9eb3f6.md

```diff
@@ -149,10 +208,25 @@ This `img` element has an [accessible name][] which is not equivalent to the fil
 </html>
 ```
 
+#### Inapplicable Example 5
+
+This `img` element has an empty [accessible name][].
+
+```html
+<html lang="en">
+	<img src="?" aria-label="" />
```

Comment:
I don't quite understand this example. It seems like this might fail SC 1.1.1; aria-label won't cause it to get ignored by AT.

Reply:
Yes, you're right. This example was pointed out in https://github.com/act-rules/act-rules.github.io/issues/465#issuecomment-477103962 to explain why the rule should ignore empty accessible names. But it is indeed failing the SC and thus should not be here…